src/Ui/src/context/userContext.tsx

```diff
@@ -1,21 +1,20 @@
-import { createContext, useState, useEffect } from "react"
+import { createContext, useState } from "react"
 import { Alert } from "react-native"
-import { Position, User, UserUseCase } from "@domain/entities"
+import { ListUserParams, Position, User, UserUseCase } from "@domain/entities"
 import { GitRepository, GitUser } from "@infrastructure/dto"
 import { HttpRepository } from "@domain/repositories"
-
+import { geohashForLocation } from "geofire-common";
```

Comment:
Hey, maybe it's better to create a helper to this function and inject it into the context, this way it's going to be easier to mock when testing

Reply:
Criei um helper e agora estou passando como prop para dentro do Provider